### DISCUSSION TITLE
Assuntos para Eventos

### DISCUSSION THREAD

**Discussion by willianjusten** [frontendbr/forum#94] [Discussão]
Pegando o bonde do [tweet do Jean Carlo Emer](https://twitter.com/jcemer/status/710618827772579840), queria saber que assuntos a galera tá buscando em eventos =D

**Comment by mtmr0x** [comment]
Nós no [FrontUX](http://www.meetup.com/pt-BR/FrontUX/) usamos o Typeform para que os proprios participantes mandem sugestões de temas e filtramos as melhores opções. Tem dado resultados muito bons!

**Comment by willianjusten** [comment]
Passa para cá também @matmarsiglio, o Typeform é bem legal, mas assim os outros não conseguem ver o que já foi pedido e aí acaba repetindo muito. Aqui fica aberto a todos, acho mais bacana =D

**Comment by letanure** [comment]
o que vejo como bons tópicos
1. [graphql](https://facebook.github.io/graphql/)
2. [relay](https://facebook.github.io/relay/)
3. flux etc
4. quando mudar para ES6, como e porque
5. overview de novas tecnologias e esse monte de framework, o que fazer afinal?
6. arquitetura de aplicacoes grandes, modules
7. polymer, pra icentivar
8. direcionamento de carreira no geral

**Comment by felipefialho** [comment]
Muito bom os tópicos do @letanure, são coisas que não tenho visto muito.

Coloco ainda:
- PostCSS
- Organização de diretórios (essencial, sério!)
- SVG e SMIL

**Comment by mtmr0x** [comment]
@willianjusten realmente a informação não está aberta dos dados que recebo, mas eu vou abrir pra ver se consigo organizar uma planilha.

A lista não é grande e tem muitos bem parecidos, então vou filtrar pra compartilhar! :smile:

**Comment by willianjusten** [comment]
@matmarsiglio, show de bola! Fico no aguardo! \o/

**Comment by DevCarlosAugusto** [comment]
@LFeh, sobre SVG + SMIL, acho que já é algo que não tem tanto suporte né ?.....não sei o estado atual, sei que a microsoft continua evitando dar suporte ao SMIL, nesse link ([SMIL](http://alistapart.com/article/web-animation-past-present-and-future....)) explica melhor sobre uma nova API de animação, mas respondendo ao  tópico, acho que deveriam ter mais coisas sobre SVG por ai mesmo....

**Comment by tcelestino** [comment]
- Redux, Flux...
- PostCSS
- Flexbox
- Atomic Design
- SVG
- Performance (ferramentas, técnicas)
- Browsers DevTools

**Comment by clarabatt** [comment]
- SQLxNoSQL: Não um em detrimento do outro, mas suas diferenças e em que tipo de projeto cada um se encaixa melhor.
- NPM como task runner
- Ambientes de desenvolvimento Vagrant/Docker
- Escalabilidade de projetos

**Comment by yanmagale** [comment]
Gosto desta ideia do @matmarsiglio em usar o Typeform....

Acredito que isto varia de evento para evento, tamanho, meetup, público alvo e etc.Agora eu não me recordo o nome, mas existe um meetup aqui em BH que a galera pode submeter talks e ai existe uma votação entre os membros, as mais votadas fazem àquela edição do evento.  

Sobre assuntos em si, acho que já foi muito bem dito nos comentários anteriores.

**Comment by jonasporto** [comment]
Bacana, @YanMagale para a galera submeter talks e rolar votação, tem o [Call4Paperz](http://call4paperz.com) é [open source](https://github.com/call4paperz/call4paperz) e toda contribuição é bem vinda.
